aiida/schedulers/plugins/test_slurm.py

```diff
@@ -46,43 +54,46 @@ def test_parse_common_joblist_output(self):
 
         job_list = scheduler._parse_joblist_output(retval, stdout, stderr)
 
+        def get_job(id):
+            try:
```

Comment:
Probably be a bit easier and more efficient to just turn `job_list` into a mapping on `job_id` and then index that in the tests instead of doing `get_job`

Reply:
done